src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -282,22 +305,34 @@ class adminLogs extends viewModelBase {
             this.pauseLogs();
             return;
         }
-        
-        data = data.trim();
-        
-        if (!this.headerSeen) {
-            this.headerSeen = true;
-            return;
+
+        let customizedMsg = false;
+        if (this.connectionJustOpened()) {
+            this.connectionJustOpened(false);
+            // replace the initial 'headers' msg
+            data = "Connection established";
+            customizedMsg = true;
         }
-        
-        this.allData.push(data);
-        this.pendingMessages.push(data);
-        
+
+       this.addMessage(data.trim(), customizedMsg);
+
         if (!this.appendElementsTask) {
             this.appendElementsTask = setTimeout(() => this.onAppendPendingMessages(), 333);
         }
     }
     
+    private addMessage(msg: string, customizedMsg: boolean = false) {
+        if (customizedMsg) {
```

Comment:
why we need customizedMsg switch here? Can we avoid that and have methods:
addMEssage(msg: string)
addStudioMEssage(msg: string) <-- and this method prepends string with date and text described above and then call addMessage?

Reply:
done